packages/sdk-codegen-scripts/src/utils.ts

```diff
@@ -97,16 +117,24 @@ export const doArgs = (args: string[]) => {
         case '--help':
           generatorHelp()
           break
+        case '-n':
+        case '--nostreams':
+          noStreams = true
+          break
         default:
           {
             const values = arg.split(',').filter((v) => v.trim())
-            values.forEach((v) => {
-              const gen = findGenerator(v.trim())
-              if (gen) {
-                // Valid language match
-                langs.push(gen.language)
-              }
-            })
+            if (values[0] !== 'all') {
```

Comment:
I'm a little lost about how `'all'` comes into play (both here and L103)

Reply:
it's possible someone may do, for example:

```sh
yarn -n all
```
for example

Reply:
I could probably remove the premature optimization bit of 103, but it's such a corner case ...